main.html

```diff
@@ -161,36 +163,26 @@
         <div class="tab_container">
             <div id="tabs">
                 <ul class="mode-disconnected">
-                    <li class="tab_landing"><a href="#" i18n="tabLanding" class="tabicon ic_welcome"
-                        title="Welcome"></a></li>
-                    <li class="tab_help"><a href="#" i18n="tabHelp" class="tabicon ic_help"
-                        title="Documentation & Support"></a></li>
-                    <li class="tab_firmware_flasher"><a href="#" i18n="tabFirmwareFlasher"
-                        class="tabicon ic_flasher" title="Firmware Flasher"></a></li>
+                    <li class="tab_landing"><a href="#" i18n="tabLanding" class="tabicon ic_welcome" title="Welcome"></a></li>
+                    <li class="tab_help"><a href="#" i18n="tabHelp" class="tabicon ic_help" title="Documentation & Support"></a></li>
```

Comment:
never mind, was just a formatting in your PR.  I will fix via another commit after merging.